ci/Dockerfile.fedora

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG DISTRO_VERSION=27
+FROM fedora:${DISTRO_VERSION}
+MAINTAINER "Carlos O'Ryan <coryan@google.com>"
+
+RUN dnf makecache
+RUN dnf install -y \
+    autoconf \
+    automake \
+    c-ares-devel \
+    clang \
+    cmake \
+    curl \
+    doxygen \
+    gcc-c++ \
+    git \
+    golang \
+    lcov \
+    libtool \
+    make \
+    openssl-devel \
+    pkgconfig \
+    python \
+    shtool \
+    unzip \
+    wget \
+    zlib-devel
+
+ARG NCPU=2
+ARG CXX=g++
+ARG CC=gcc
+
+# ... capture the Travis job number, effectively this busts the cache
+# in each build, which we want anyway ...
+ARG TRAVIS_JOB_NUMBER=""
+RUN echo Running build=${TRAVIS_JOB_NUMBER}
+
+# We assume that this is running on a (clean-ish) checkout of
+# google-cloud-cpp, including submodules, and copy the files to a
+# directory where we will build ...
+COPY . /var/tmp/build/gccpp
+
+WORKDIR /var/tmp/build/gccpp/.build
+RUN cmake ..
+RUN make -j ${NCPU}
```

Comment:
Do you want to be explicit and run "make ... all" or is the `all` target the default, always?

I believe in `Makefile` the default target is the first one listed, not `all` necessarily (see https://stackoverflow.com/a/2057716).

Reply:
Done.